utils/k6_prepare_tracer.py

```diff
@@ -0,0 +1,253 @@
+                        "recipient_balance_before": f"{recipient_balance_before}",
+                        "recipient_balance_after": f"{erc20_wrapper.get_balance(account_receiver)}",
+                        "amount": transfer_amount
+                    })
+            except Exception as e:
+                print(f"Error in erc20_transfer_data: {e}") 
+
+    def storage_contract_data(self, calls_number):
+        print("Storage contract deployment...")
+        contract, _ = self.web3_client.deploy_and_get_contract(
+        "common/StorageSoliditySource",
+            "0.8.8",
+            self.get_account(),
+            contract_name="Storage",
+            constructor_args=[],    
+        )
+        
+        storage_contract = StorageContract(self.web3_client, contract)
+
+        print("Storage contract calls...")
+        for i in range(calls_number):
+            try:
+                sender_account = self.get_account()
+                store_value = random.randint(0, 100)
+                
+                tx_obj, _, receipt = storage_contract.call_storage(sender_account, store_value, "blockNumber")
+                
+                print(f"Storage contract call {i}, receipt status: ", receipt["status"])
+                self.historical_data["storage_contract_calls"].append({
+                        "blockHash": receipt["blockHash"].hex(),
+                        "blockNumber": hex(receipt["blockNumber"]),
+                        "storage_contract_address": storage_contract.contract_address,
+                        "sender": sender_account.address,
+                        "store_value": store_value,
+                        "retreive_function_tx": tx_obj,
+                        "tx_hash": receipt["transactionHash"].hex()
+                    })
+            except Exception as e:
+                print(f"Error in call storage contract functions: {e}") 
+
+    def event_caller_contract_data(self, calls_number):
+        account_owner = self.get_account()
+        
+        print("Events caller/callee contracts deployment...")
+        contract, _ = self.web3_client.deploy_and_get_contract("common/EventsCheckerCaller", 
+                                                               "0.8.15", 
+                                                               account=account_owner)
+        _, contract_deploy_tx = self.web3_client.deploy_and_get_contract("common/EventsCheckerCallee", 
+                                                                         "0.8.15", 
+                                                                         account=account_owner)
+        for i in range(calls_number):
+            try:
+                sender_account = self.get_account()
+                tx = self.web3_client.make_raw_tx(from_=sender_account)
+                instruction_tx = contract.functions.emitAllEventsAndCallContractCalleeWithEvent(
```

Comment:
this tx contains 10 solana txs, so I used it as iterative one. Is smth wrong here?

Reply:
you have 2 cases with iterative transactions with some function call
Do you need both for load tests?

Reply:
We have two request's types: by block number and by block hash, every requests in test will be doubled. 
I do following requests in debug_traceTransaction scenario:
1) call a storage contract (by blockNumber and blockHash) with our standard tracer type (a simple request)
2) call an iterative tx (by blockNumber and blockHash) with `callTracer` tracer type
3) call a contract that emits several events (by blockNumber and blockHash) with `callTracer` tracer type - This case is important because of log structure in the tracer response: it is more difficult than usual and such requests can contribute into the load